pkgs/development/python-modules/sip5/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchPypi, buildPythonPackage, packaging, toml }:
+
+buildPythonPackage rec {
+  pname = "sip5";
```

Comment:
just `sip`

Reply:
Done, I also moved the nix expression to `sip/5.x.nix`.